packages/api/database/seeds/4_TechnologySeeder.js

```diff
@@ -40,57 +40,58 @@ class TechnologySeeder {
 
-		for (const technology of technologies) {
-			/** Create KEYWORDS in Technologies */
-			const keywordTerms = await Factory.model('App/Models/Term').createMany(5);
-			await keywordTaxonomy.terms().saveMany(keywordTerms);
-			await technology.terms().attach(keywordTerms.map((keywordTerm) => keywordTerm.id));
-
-			/** Create a CLASSIFICATION in Technology */
-			const classificationTerm =
-				classificationTerms.rows[
-					Math.floor(Math.random() * classificationTerms.rows.length)
-				];
-			await technology.terms().attach([classificationTerm.id]);
-
-			/** Create a STAGE in Technology */
-			const stageTerm = stageTerms.rows[Math.floor(Math.random() * stageTerms.rows.length)];
-			await technology.terms().attach([stageTerm.id]);
-
-			/** Create a DIMENSION in Technology */
-			const dimensionTerm =
-				dimensionTerms.rows[Math.floor(Math.random() * dimensionTerms.rows.length)];
-			await technology.terms().attach([dimensionTerm.id]);
-
-			/** Create a CATEGORY in Technology */
-			const categoryTerm =
-				categoryTerms.rows[Math.floor(Math.random() * categoryTerms.rows.length)];
-			await technology.terms().attach([categoryTerm.id]);
-
-			/** Create a SUBCATEGORY in Technology */
-			const subCategoryTerms = await Taxonomy.getTaxonomyTerms('CATEGORY', categoryTerm.id);
-			const subCategoryTerm =
-				subCategoryTerms.rows[Math.floor(Math.random() * subCategoryTerms.rows.length)];
-			await technology.terms().attach([subCategoryTerm.id]);
-
-			/** Create a TARGET_AUDIENCE in Technology * */
-			const targetAudienceTerm =
-				targetAudienceTerms.rows[
-					Math.floor(Math.random() * targetAudienceTerms.rows.length)
-				];
-			await technology.terms().attach([targetAudienceTerm.id]);
-
-			/** Create a BIOME in Technology * */
-			const biomeTerm = biomeTerms.rows[Math.floor(Math.random() * biomeTerms.rows.length)];
-			await technology.terms().attach([biomeTerm.id]);
-
-			/** Create a GOVERNMENT_PROGRAM in Technology * */
-			const governmentProgramTerm =
-				governmentProgramTerms.rows[
-					Math.floor(Math.random() * governmentProgramTerms.rows.length)
-				];
-			await technology.terms().attach([governmentProgramTerm.id]);
-		}
+		const getRandom = (taxonomyTerms) => {
+			const result =
+				taxonomyTerms.rows[Math.floor(Math.random() * taxonomyTerms.rows.length)];
+			return result.id;
+		};
+
+		await Promise.all(
+			technologies.map(async (technology) => {
+				/** Create a CLASSIFICATION in Technology */
+				const classificationTerm = getRandom(classificationTerms);
+
+				/** Create a STAGE in Technology */
+				const stageTerm = getRandom(stageTerms);
+
+				/** Create a DIMENSION in Technology */
+				const dimensionTerm = getRandom(dimensionTerms);
+
+				/** Create a CATEGORY in Technology */
+				const categoryTerm = getRandom(categoryTerms);
+
+				/** Create a SUBCATEGORY in Technology */
+				const subCategoryTerm = getRandom(categoryTerms);
```

Comment:
@NaylsonFerreira eu percebi um pequeno erro aqui. No caso da taxonomia de categoria você precisa primeiro sortear uma categoria pai e depois carregar as categorias filhas e sortear uma dentre elas. Favor faça essa alteração e teste.

```suggestion
				const subCategoryTerms = await Taxonomy.getTaxonomyTerms('CATEGORY', categoryTerm.id);
				const subCategoryTerm = getRandom(subCategoryTerms);
```